docs/BUILD.md

```diff
@@ -0,0 +1,69 @@
+# Building Kubicorn
+
+## Make
```

Comment:
I think this is a good idea, its there for docker so it should definitely be here for golang.